test/controllers/EditSetControllerSpec.scala

```diff
@@ -724,16 +662,9 @@ class EditSetControllerSpec extends BaseSpec {
             title = "Browse Edit Set (Page 1 of 2)",
             caption = "Showing 1 - 10 of 12 records",
             header = "Edit set: COAL 80 Sample",
-            button = ExpectedActionButton("reorder", "Sort edit set"),
-            expectedOptionsForField = Seq(
-              ExpectedSelectOption(FieldNames.ccr, "CCR", selected = true),
-              ExpectedSelectOption(FieldNames.scopeAndContent, "Scope and Content"),
-              ExpectedSelectOption(FieldNames.coveringDates, "Covering Dates")
-            ),
-            expectedOptionsForDirection = Seq(
-              ExpectedSelectOption(orderDirectionAscending, "Ascending", selected = true),
-              ExpectedSelectOption(orderDirectionDescending, "Descending")
-            ),
+            ccrTableHeader = ExpectedTableHeader("ccr", "descending"),
```

Comment:
The names don't seem to matter; I changed all to `xxx` and it passed.

Reply:
Fixed, matchers did not work the way I thought they did.

Reply:
Yeah - it was only returning the second.

Confirmed.